web-jehlomat/src/Components/Navigation/ModalTitleBar.tsx

```diff
@@ -0,0 +1,28 @@
+import { FC, MouseEventHandler } from 'react';
+import * as s from './ModalTitleBarStyles';
+import { NavigationTitle } from '../../utils/typography';
+import { IconButton } from '@mui/material';
+
+interface ITitleBar {
```

Comment:
In components before this one, you use interfaces without `I` as a prefix, but this uses the `I` prefix. Please be consistent, personally, for interface props, I add `Props` postfix.